src/projections/updateProjection.ts

```diff
@@ -8,10 +8,10 @@ import { debug, convertToCommandError } from "../utils";
 
 export interface UpdateProjectionOptions extends BaseOptions {
```

Comment:
**COMMENT:** To have RFC fully implemented, `TrackEmittedStreams` has to be added.

Reply:
Indeed, this PR only covers the parts of the RFC currently supported by the server over GRPC (`TrackEmittedStreams` is missing in the protos). I should probably update the wording of the PR to mention that (the decision to "polyfill" the gaps was made after this PR).